helm/hpcc/templates/_helpers.tpl

```diff
@@ -1244,6 +1244,50 @@ dali data
 {{- end -}}
 {{- end -}}
 
+{{/*
+A template to generate the standard app.kubernetes.io labels
+
+root name(k8s application name) component(component within the application, can be same as app) instance 
+
+https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
+----------------------------+-------------------------------------------------------------+----------
+| Label                     | Description                                                 | Example |
+----------------------------+-------------------------------------------------------------+----------
+app.kubernetes.io/name       The name of the application                                   mysql
+app.kubernetes.io/component  The component within the architecture                         database
+app.kubernetes.io/instance   A unique name identifying the instance of an application      mysql-abcxzy
+app.kubernetes.io/version    The current version of the application                        5.7.21
+app.kubernetes.io/part-of    The name of a higher level application this one is part of    wordpress
+app.kubernetes.io/managed-by The tool being used to manage the operation of an application helm
+app.kubernetes.io/created-by The controller/user who created this resource 	               controller-manager
+helm.sh/chart                This should be the chart name and version
+*/}}
+{{- define "hpcc.addStandardLabels" }}
+app.kubernetes.io/part-of: HPCC-Platform
+{{- if .name }}
+app.kubernetes.io/name: {{ .name }}
+{{- end }}
+{{- if .component }}
+app.kubernetes.io/component: {{ .component }}
+{{- end }}
+{{- if .instance }}
+app.kubernetes.io/instance: {{ .instance }}
+{{- end }}
+{{- if .root }}
+ {{- if hasKey .root "Release" }}
+app.kubernetes.io/managed-by: {{ .root.Release.Service }}
+ {{- end }}
+ {{- if hasKey .root "Chart" }}
+  {{- if .root.Chart.Version }}
+app.kubernetes.io/version: {{ .root.Chart.Version }}
+   {{- if .root.Chart.Name }}
+helm.sh/chart: {{ .root.Chart.Name }}-{{ .root.Chart.Version | replace "+" "_" }}
+   {{- end }}
```

Comment:
based on the other commens/conv., I suspect better to revert helm.sh/chart to way it was in each template file.

Reply:
you're right, it was about "helmVersion" - my comment is misplaced.